tooling/nargo_cli/src/cli/check_cmd.rs

```diff
@@ -110,9 +107,7 @@ fn check_package(
                 eprintln!("Note: Prover.toml already exists. Use --overwrite to force overwrite.");
             }
 
-            let any_file_written = should_write_prover;
-
-            Ok(any_file_written)
+            Ok(())
```

Comment:
Based on https://github.com/noir-lang/noir/pull/7170/files#diff-fe0ef4227fae4e8c5894bbc6bdc9ddaa5b43a5f8a7022e75244b525f817de47c I thought this was somehow an indication for the fact that the `Prover.toml` file has been created. Should there be a dedicate feedback about it in `check_package` instead, now that `any_file_written` is no longer returned?

Reply:
I don't think that we need to display any feedback for this file being created tbh.

Reply:
Okay. I wasn't sure if it was common knowledge that `check` creates this template, and thought it would be good to give the user a hint in case they were wondering where a new file came from, but I suppose they will know once it starts warning them about the `--overwrite` flag.